libraries/ESP8266WiFi/src/ESP8266WiFiMulti.h

```diff
@@ -58,6 +60,12 @@ class ESP8266WiFiMulti {
 
     private:
         WifiAPlist APlist;
+        unsigned long connect_timeout_ms;
+
+        void selectBest(BestNetwork& best, int8_t scanResult);
+        void handleConnectComplete(wl_status_t status);

```

Comment:
This is a debug method, and there is also no need for it to be part of the class, or exported in the .h. Please remove and make the function a static free standing function in the .cpp.